incubator-tools/tagging_customer_account_number/tagging_customer_account_number.ipynb

```diff
@@ -0,0 +1,336 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #70.        return entity
</pre><p>Similar to a previous notebook, the entity should be initialized with all values instead of assigning values after creation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/682/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>